.github/templates-list-validator/index.js

```diff
@@ -0,0 +1,71 @@
+const github = require('@actions/github');
+const core = require('@actions/core');
+const { readFileSync } = require('fs');
+
+async function run() {
+  try {
+    const token = process.env.GITHUB_TOKEN || core.getInput('token', { required: true });
+    //TODO we should not only check the content of the readme but also the content of the website listing from https://github.com/asyncapi/website/blob/master/content/docs/tooling.md
+    const templatesListContent = getReadmeContent();
+    const officialTemplates = await searchOfficialTemplates(token);
+
+    const missingTemplates = officialTemplates.filter(str => templatesListContent.indexOf(str) === -1);
```

Comment:
The following suggestions are much more readable in my opinion:

```suggestion
    const missingTemplates = officialTemplates.filter(str => templatesListContent.includes(str) === false);
```

or

```suggestion
    const missingTemplates = officialTemplates.filter(str => !templatesListContent.includes(str));
```

But these are just tastes :)

Reply:
I like the 2nd one, it is clean, didn't know -1 evaluates to false, makes sense

Reply:
It doesn't evaluate to false. I changed the method to `includes`, which returns a boolean.

Reply:
🤦